src/scss/components/MainNavButton/MainNavButton.scss

```diff
@@ -14,7 +14,7 @@
 
   .pip {
     position: absolute;
-    top: 14px;
+    top: 13px;
```

Comment:
I'm assuming these are `px` for good reason?

Reply:
Im not sure, there might have been. I saw it wasn't aligning properly, so I fixed it